wherehows-backend/app/controllers/Application.java

```diff
@@ -76,4 +81,37 @@ public static Result index() {
   public static Result healthcheck() {
     return ok("GOOD");
   }
+
+  public static Result deploylibsbackend() {
+    String env = System.getenv("WHZ_APP_HOME");
```

Comment:
Make this 'private static final' at the top and name WHZ_APP_HOME.
Both frontend and backend.